app/models/katello/host_collection.rb

```diff
@@ -60,8 +61,14 @@ def errata(type = nil)
       type ? query.of_type(type) : query
     end
 
-    def total_hosts
-      hosts.length
+    def cache_key
+      "#{self.class.name}/#{self.id}"
+    end
+
+    def total_hosts(force_update_cache: true)
+      Rails.cache.fetch("#{cache_key}/total_hosts", expires_in: 1.minute, force: force_update_cache) do
```

Comment:
In a few other places in Katello we have the argument as `cached` with a default of `false`, and then we pass `!cached` to `Rails.cache.fetch`. Prob better to stay consistent with that.

https://github.com/Katello/katello/blob/f3891b2b8dacb2f5f10e19b5df22df330019aff2/app/models/katello/concerns/organization_extensions.rb#L103-L104

Reply:
I saw two standards, so I was following this one: https://github.com/Katello/katello/blob/master/app/models/katello/host/content_facet.rb#L315

But `cached` is less verbose, so I like that.